extensions/functions_geometry.yaml

```diff
@@ -0,0 +1,39 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "st_point"
+    description: >
+      Return a point with the given x and y coordinate values.
+    impls:
+      - args:
+          - name: x
+            value: fp32
```

Comment:
Could you create a point with integers or fp64?  How about decimal?  Note that the point structure currently is made up of i32.

Reply:
oops, those should be integers actually